source/standards/securing-services.html

```diff
@@ -0,0 +1,25 @@
+---
+title: Securing services
+expires: 2017-10-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+At GDS we are committed to iterating our security initiatives and are determined to make them an exemplary and active standard.
+
+## Our core security teams are:
+
+* Security Operations - Monitoring and analysis
+* Security Engineering - Proactive defence
+* Security Architecture - Cross-government cooperation
+
+## We are currently working on:
+
+* threat intelligence - working within the industry and intelligence community to constantly collect and process intelligence in order to develop an accurate threat profile, this includes open source collection
```

Comment:
I'm not sure this is a standard?  This documents the structure of an organisation, rather than sets out how we build or operate a service.